### PR TITLE
Updating README to reflect current progress

### PR DESCRIPTION
**Pull request by kenazk**
This commit updates the README to reflect our current progress and
roadmap. It also adds some of the relevant K8s artifacts to the repo so
it's easier to demo. It changes the default "Getting Started" example to
a real Workflow now that we have delete functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +22,28 @@ For a more detailed view of how we think about Lyra, check out our introductory
...
 
-There is an AWS Workflow called `attach` in `plugins\attach.pp` (To run: ` $ ./build/lyra --debug apply attach`). This will use the default AWS region configured in `~/aws/config` and create real AWS resources.  To delete the Workflow (i.e. its resource), run ` $ ./build/lyra --debug delete attach`.  The `data.yaml` file can be used to configure AWS tag data.
+> **!! WARNING: THIS WORKFLOW CREATES REAL RESOURCES ($$) !!**
 
-## Integrating with Kubernetes
-1. Install the Lyra CRD as described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
-2. Start Lyra in controller mode: ` $ ./build/lyra --debug controller`
-3. Create a Workflow resource, also described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
+1. Run the binary with the [sample Workflow](https://github.com/lyraproj/lyra/blob/master/plugins/attachyaml.yaml): ` $ ./build/lyra apply awsyaml --debug`
+2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete attach --debug`.
```
Comment (markfuller) [inline README.md]: delete `awsyaml` rather than `attach`

File: plugins/attachyaml.yaml [hunk 2]
```diff
@@ -1,9 +1,5 @@
-attachterraformyaml:
+attachyaml:
```
Comment (markfuller) [inline plugins/attachyaml.yaml]: attach is an out of date name (based on attaching internet gateways), is it worth renaming all of the workflows to aws (or awsinstance), aws2, etc for example?

Comment (jdwelch) [reply plugins/attachyaml.yaml]: Something descriptive like "aws_vpc_ec2_sample" would be preferable.

File: README.md (hunk 1 above)
Comment (jdwelch) [reply README.md]: +1; see also naming comment below

File: README.md [hunk 3]
```diff
@@ -22,20 +22,28 @@ For a more detailed view of how we think about Lyra, check out our introductory
...
+> **!! WARNING: THIS WORKFLOW CREATES REAL RESOURCES ($$) !!**
 
-## Integrating with Kubernetes
-1. Install the Lyra CRD as described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
-2. Start Lyra in controller mode: ` $ ./build/lyra --debug controller`
-3. Create a Workflow resource, also described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
+1. Run the binary with the [sample Workflow](https://github.com/lyraproj/lyra/blob/master/plugins/attachyaml.yaml): ` $ ./build/lyra apply awsyaml --debug`
+2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete attach --debug`.  
+
+This workflow is an AWS Workflow called `attachyaml` in `plugins\attachyaml.yaml`. This will use the default AWS credentials configured in your `~/.aws/config`. Region is currently hard-coded to `eu-west-1`.
```
Comment (jdwelch) [inline README.md]: Needs `~/.aws/credentials` (See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html); `~/.aws/config` doesn't seem to matter for this workflow

File: README.md [hunk 4]
```diff
@@ -22,20 +22,28 @@ For a more detailed view of how we think about Lyra, check out our introductory
...
-1. Install the Lyra CRD as described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
-2. Start Lyra in controller mode: ` $ ./build/lyra --debug controller`
-3. Create a Workflow resource, also described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
+1. Run the binary with the [sample Workflow](https://github.com/lyraproj/lyra/blob/master/plugins/attachyaml.yaml): ` $ ./build/lyra apply awsyaml --debug`
+2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete attach --debug`.  
+
+This workflow is an AWS Workflow called `attachyaml` in `plugins\attachyaml.yaml`. This will use the default AWS credentials configured in your `~/.aws/config`. Region is currently hard-coded to `eu-west-1`. 
+
+
+### Deploying Workflows with Kubernetes
```
Comment (jdwelch) [inline README.md]: Needs something about building and running the operator

Comment (kenazk) [reply README.md]: Operator should be auto pulled into the build: 

https://github.com/lyraproj/lyra/blob/master/cmd/lyra/cmd/controller.go#L12

I don't think there's anything else needed @scottyw?

File: README.md (hunk 3 above)
Comment (kenazk) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (kenazk) [reply README.md]: It's opposite. The controller works in a pull model from the cluster. The controller subscribes to 'Workflow' events created on the cluster. When starting the controller, it pulls the same kubeconfig that `kubectl` uses to figure out which k8s cluster to start listening to. Registering the CRD is what allows the cluster to understand what a 'Workflow' is. 

It's a pretty brilliant model the K8s community came up with that allows you to remotely dev your controller, but subscribe to a "production" k8s cluster. It's also what I'm noodling on in this proposal:
https://github.com/kenazk/lyra/blob/proposal/docs/lyra-controller-proposal/proposal.md

File: plugins/attachyaml.yaml (hunk 2 above)
Comment (kenazk) [reply plugins/attachyaml.yaml]: renamed

File: README.md (hunk 1 above)
Comment (kenazk) [reply README.md]: renamed

Comment (kenazk) [reply README.md]: Fudge. I missed this one.

File: plugins/aws_vpc_yaml.yaml [hunk 5]
```diff
@@ -22,15 +13,27 @@ yamltest:
         enable_dns_support: false
         is_default: false
         state: available
-        tags: $tags
```
Comment (kenazk) [inline plugins/aws_vpc_yaml.yaml]: Adding inline until we have a fix for #76

**Pull request closed without merging**